content/en/core/releases/_index.md

```diff
@@ -22,7 +22,8 @@ It is recommended that all projects update regularly multiple times a year to ge
 
 | Version | Status    | Release date | End of life |
 |---------|-----------|--------------|-------------|
-| 4.13.x  | Supported | 22-Oct-2024  | TBA         |
+| 4.13.x  | Supported | 31-Oct-2024  | TBA         |
+| 4.13.x  | Supported | 22-Oct-2024  | 22-Feb-2025 |
```

Comment:
The logic here is confusing but the 4.13.x end of life is 3 months after 4.14.0 is released, _not_ 3 months after 4.13.x is released. Because there is no 31st of Feb I've pushed it to 1st of March.

```suggestion
| 4.13.x  | Supported | 22-Oct-2024  | 01-Mar-2025 |
```